### PR TITLE
[Project Submission] Creatorr

### PR DESCRIPTION
**Pull request by demo-hub**

### CODE REVIEW

File: projects/creatorr.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- User experience is key
+- Front end dev
+
+Nuno Monteiro, Front-end dev
+- Ability to quickly change gears and adapt to changes
+- Creative thinking
+- Out of the box ideas
+
+## How the community can engage
+Email: - info@creatorr.fan
```
Comment (autonome) [inline projects/creatorr.md:70]: Hey! can you add a show-and-tell topic in the discussions forum? thanks!

https://github.com/ipfs/community/blob/master/README.md#ecosystem-projects

**Review by autonome — CHANGES_REQUESTED** [review]

File: projects/creatorr.md (hunk 1 above)
Comment (demo-hub) [reply projects/creatorr.md:70]: Done

Comment (autonome) [reply projects/creatorr.md:70]: Can you link the discussion here?

File: projects/creatorr.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## Category 
+NFT, Dapp, IPFS hosting, Social Media, Streaming
+
+## Project Description
+The goal is for Creatorr to be a platform which helps create an incentivized content creation support system. Creatorr fans earn unique dynamic NFT tokens from their content creator in exchange for donations and support. The tokens are collectible, tradeable and/or sellable.
+
+## Project Status
+MVP functionality being developed. Next Milestone is private alpha testing.
+
+## Previews
```
Comment (autonome) [inline projects/creatorr.md:13]: Can you add a screenshot here?

**Review by autonome — CHANGES_REQUESTED** [review]
Couple small changes

File: projects/creatorr.md (hunk 2 above)
Comment (demo-hub) [reply projects/creatorr.md:13]: Added 2 screenshots

File: projects/creatorr.md (hunk 1 above)
Comment (autonome) [reply projects/creatorr.md:74]: Just need this linked here

**Review by autonome — CHANGES_REQUESTED** [review]
needs discussion link

File: projects/creatorr.md (hunk 1 above)
Comment (demo-hub) [reply projects/creatorr.md:75]: Sorry, I linked it now.

**Comment by autonome** [comment]
Looks good thanks!

**Pull request merged**